packages/elements/src/datetime-picker/index.ts

```diff
@@ -1159,13 +1178,40 @@ export class DatetimePicker extends ControlElement implements MultiValue {
     ></ef-time-picker>`;
   }
 
+  /**
+   * Create calendar slot
+   * @param id Calendar identifier
+   * @returns calendarSlots slots that will cascade to calendar
+   */
+  private createCalendarSlots(id: string): HTMLSlotElement[] | null {
+    if (!this.opened) {
+      return null;
+    }
+
+    const querySlots = Array.from(this.querySelectorAll('[slot]'));
+    return querySlots
+      .filter((slot) => {
+        const isToSlot = id === 'calendar-to' && slot.slot.startsWith('to-');
+        const isFromSlot = id === 'calendar' && slot.slot.startsWith('from-');
+        const isISODateSlot = id === 'calendar' && /^-?\d{1,6}(-\d{2}(-\d{2})?)?$/.test(slot.slot);
+        return isToSlot || isFromSlot || isISODateSlot;
```

Comment:
`to-` or `from-` should be filtered out while `to-2023` should remain, right?
```suggestion
        return isToSlot || isFromSlot && isISODateSlot;
```

Reply:
Done